docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>Same problem with table than above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Now it looks well... I don't know what's going on</p>

Reply:
<p>It looks ok on my notebook and when I render the docs locally</p>